### DISCUSSION TITLE
Recommended Retail Price (RRP)

### DISCUSSION THREAD

**Discussion by ghost** [PrestaShop/PrestaShop#40156] [Ideas]
Hi.
Please add native RRP price to 1.7
http://nemops.com/prestashop-rrp-price/

Like NIKE CATALOG PRICE is 499 (manufacturer suggested retail price)
Our PRICE is 399

Thanks!

**Comment by ghost** [comment]
One of my customer also want to make catalog rules based on that RRP price so this field should be integrated in pricing logic.

**Comment by mariem-abid** [comment]
@colinegin, @sarahdib, what do you think?

**Comment by ghost** [comment]
Hello @PrestaShark,

Thanks for the feedback, we'll study it 👍 

Can you give me more info about the cart rule your customer wants to create?

**Comment by ghost** [comment]
Hi @samuel-pires thanks for Your time.
It's quite "simple". He wants to create catalog discount based on that RRP price (manufacturer price) instead of prestashop final price.

So RRP is 499zł and he want to be able to make 30% discount from this price to get 349zł.
This kind of catalog rule will be **VERY** usefull for retailers to manage seasonal SALE offers. 

Also this rule should have top priority above Specific Prices when created.

I hope this help You understand :)

**Comment by ghost** [comment]
Main problem in many stores is that "Final Price" is "floating" price. 
At begining that price is near "manufacurers price" but in time this price is manually changed by staff (altered).

When they want to create sale offers then sometimes sales below price may occour... Retail Price will be savior in that cases IMO.

**Comment by ghost** [comment]
![discount](https://user-images.githubusercontent.com/8243273/61285587-81387b80-a7c1-11e9-9e57-8bf718f74e35.png)

**Comment by Hlavtox** [comment]
@PrestaShark Since our company uses Prestashop, I always added RRP price to the code manually. I just made it working just fine in 1.7.6, but I don't know about cart rules, that would need much deeper modification...

@samuel-pires @mariem-abid  I also join PrestaShark on this. Would be a nice feature to have by default.

![RRP](https://user-images.githubusercontent.com/6097524/61287488-433d5680-a7c5-11e9-9ea3-7627cf14b79b.PNG)

Another stuff for you guys to think about, I was also missing these fields in default Prestashop, which could be helpful to a lot of merchants:
- Manufacturer reference. We have our reference, then you have EAN etc., but products also have a reference set by the manufacturer, by which users search it on the web.
- Package contents. To show what accessories are inside the package. (something like short_description)

**Comment by ghost** [comment]
Thank you for all the info.

@PrestaShark,
Just to be sure. You customer wants to be able to set the product price - let's call it "Final price" - and the "RRP price" when creating a product.
Then during sales periods, he/she would like to create a catalog rule on some products but to apply the discount on the "RRP price" and not on the "Final price". Am I correct?
If so, why would he/she apply it to the "RRP price" and not the "Final price"?


@Hlavtox, an issue is currently in our 1.7.7 backlog regarding the Manufacturer Part Number #11517.
I created another issue about the package content so we can discuss it there and track it separately. Please feel free to add more details in it #14681   👍

**Comment by ghost** [comment]
For marketing trick like "30% off*"

*from manufacturer recommended retail price ;)
That kind of sale price management would be very efficient like in image example above.
Final price sometimes "floating in time" (altered by staff). Retail price would be something like "fixed price" and creating catalog rules by this price will never impact on sales below costs.

Of course RRP price is not mendatory and this may cause problems in pricing/catalog rule logic.

Please consider that new native field and @Hlavtox please make Pull Request with that modiffication to Prestashop repository to add speed in that implementation in near future.

Next step after is integrate that field to pricing rules.

THANKS!

**Comment by Hlavtox** [comment]
Just a tip for FO design. This is how I did it. :-)

![frontend](https://user-images.githubusercontent.com/6097524/69348372-6053cb80-0c76-11ea-8710-65b87ca9fa4d.JPG)

**Comment by ghost** [comment]
Well done. Now please make PR for this improvement :)

**Comment by ghost** [comment]
Friendly reminder @sam-pires  ;)

RRP price would be very useful for marketing purposes.
https://user-images.githubusercontent.com/8243273/61285587-81387b80-a7c1-11e9-9e57-8bf718f74e35.png

Making discounts very effectively

**Comment by Hlavtox** [comment]
Hi everyone, I think the era of RRP price has come to an end. The new EU directive 2019/2161 is coming in effect no later than 28.5.2022.

Basically - if showing any discount, it must be from the product price in last 30 days. Plus, the seller must be able to prove that he was really selling the products for that price.

Ping @ttoine - maybe you can explore a bit more on that topic? I was thinking, and also we should be able to set, if a specific price should be displayed as a discount or not. (With original price and X%/X$ discount)

**Comment by ttoine** [comment]
To me this must be a custom development or module, or even several modules:
- not all merchants really needs this feature
- the way it must work is different in many countries or world regions, and of course it might also change depending on markets and verticals
- how do you handle getting the information from the "official catalogs" ? distributors API ? custom services (and who host it) ? manual update ? other ?
- directives are evolving regularly, adding work on the project to maintain it

In the PrestaShop open source project, the target is to have the most important and common features for all merchants, because it is already a lot of work to maintain the software.
Note that this apply also to all open source projects.

In the past, I would have had a look at Pricing Assistant, but I see they have been acquired by Content Square. So maybe they provide something for PrestaShop that would solve your problem ?

**Comment by Hlavtox** [comment]
Well, I pinged you because I thought that PS is focusing on EU environment - following your other threads regarding compliance to EU privacy laws.

I am all good, in my case, we will just stop using the RRP price and we will be inline with the legislative. 👍

**Comment by ttoine** [comment]
Even for RGPD, it was a dedicated module and not something implemented in the Core.

**Comment by keshvathakur47** [comment]
This is best module and very chap cost  : Prestashop Recommended retail price

and display extra price 

https://modulesprestshop.com/prestashop-catalog/prestashop-recommended-retail-price-msrp-display-by-increase-base-price.html
